### ISSUE TITLE
TravisCI build dynamodb 404

### ISSUE THREAD

**Issue by nvanheuverzwijn** [kelseyhightower/confd#540]
In PR #541, the build fails because of a 404 error :
```
The command "wget -O - http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest | tar xz --directory /tmp/dynamodb" failed and exited with 2 during .
```

On my box :
```
$ wget -O - http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest
--2017-03-02 18:03:42--  http://dynamodb-local.s3-website-us-west-2.amazonaws.com/dynamodb_local_latest
Resolving dynamodb-local.s3-website-us-west-2.amazonaws.com (dynamodb-local.s3-website-us-west-2.amazonaws.com)... 52.218.160.95
Connecting to dynamodb-local.s3-website-us-west-2.amazonaws.com (dynamodb-local.s3-website-us-west-2.amazonaws.com)|52.218.160.95|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://s3-us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_latest [following]
--2017-03-02 18:03:42--  https://s3-us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_latest
Resolving s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)... 54.231.176.184
Connecting to s3-us-west-2.amazonaws.com (s3-us-west-2.amazonaws.com)|54.231.176.184|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-03-02 18:03:43 ERROR 404: Not Found.
```

Not sure if we can do anything about it..

**Comment by pedrocalais** [comment]
Same problem here.

**Comment by pedrocalais** [comment]
https://s3-us-west-2.amazonaws.com/dynamodb-local/dynamodb_local_latest.tar.gz works!

**Comment by okushchenko** [comment]
Fixed in #566
